common/app/mvt/MultiVariateTesting.scala

```diff
@@ -76,6 +76,16 @@ object ABJavascriptRenderingControl extends TestDefinition(
   def canRun(implicit request: RequestHeader): Boolean = participationGroup.contains("control")
 }
 
+object PillarCards extends TestDefinition(
+  name = "pillar-cards",
+  description = "Users in this test will see pillar cards.",
+  owners = Seq(Owner.withName("dotcom.platform")),
+  sellByDate = new LocalDate(2018, 2, 1)
+) {
+  def participationGroup(implicit request: RequestHeader): Option[String] = request.headers.get("X-GU-pillar-cards")
+  def canRun(implicit request: RequestHeader): Boolean = participationGroup.contains("true")
```

Comment:
The contains value should match the value set in Fastly. Usually we use variant/control when specifying 2 groups

Reply:
it's not really a test though, hence not using that language. it's so that individuals can see changes before they're live (i copied from the polyfill.io optin). should that still be control/variant?

Reply:
I think its okay not to have a control for now as a control wouldn't really be doing anything. But we do need a corresponding PR in fastly for the opt in.

Reply:
However, there is no reason I can see you effectively have two switches. This is a switch too. Therefore, you probably should remove the other switch and make the logic so much simpler.